false

```diff
@@ -0,0 +1,2 @@
+      invoke  active_record
+      create    db/migrate/20220114102847_add_admin_to_users.rb
```

Comment:
What does this file do? Seems to be the CLI output of creating some unrelated migration :sweat_smile: